src/components/Editor/Editor.module.css

```diff
@@ -0,0 +1,89 @@
+:global(.cm-tooltip.playground-autocompletion-dialog),
```

Comment:
You can wrap everyting in `:global`

Reply:
basically nesting? it won't work on all browsers, isn't it? only lately it became baselined

Reply:
I don't think it's even CSS nesting but CSS Modules, I mean:
```
:global {

}
```

Reply:
@YossiSaadi

Reply:
it does not work :/
so it's not even css nesting
unless I'm doing something wrong

my sheet looks something like this
```
:global {
  .cm-tooltip.playground-autocompletion-dialog {
    color: red;
  }
}
```